packages/plugin-cosmos/src/types/index.ts

```diff
@@ -0,0 +1,21 @@
+import { CosmosTransferParams } from "../actions/cosmosTransfer";
+import { assets } from "chain-registry";
+
+export type Asset = Pick<
+    (typeof assets)[number]["assets"][number],
+    "base" | "denom_units" | "display"
+>;
```

Comment:
why are you ommiting a lot of assets properties?

Reply:
I don't need them. It's easier to create mocks that fulfil interface without `as`